lib/benchee/formatters/html.ex

```diff
@@ -51,6 +53,12 @@ defmodule Benchee.Formatters.HTML do
   defp render_system_info(system, options \\ [visible: false]) do
     system_info(system, options)
   end
+  defp render_footer do
+    %{
+      benchee: Application.spec(:benchee, :vsn),
+      benchee_html: Application.spec(:benchee_html, :vsn)
+    } |> footer()
```

Comment:
I think we should rather do `footer(%{})` - a pipe with just one argument is a bit overpowered... I should maybe add credo to the build here :D